osu.Game/Overlays/BeatmapSetOverlay.cs

```diff
@@ -94,16 +79,13 @@ public BeatmapSetOverlay()
             Header.Picker.Beatmap.ValueChanged += b =>
             {
                 info.Beatmap = b.NewValue;
-
-                scroll.ScrollToStart();
+                ScrollFlow.ScrollToStart();
             };
         }
 
-        [BackgroundDependencyLoader]
-        private void load()
-        {
-            background.Colour = ColourProvider.Background6;
-        }
+        protected override OverlayHeader CreateHeader() => null;
```

Comment:
Good point

Reply:
Hmm, I'd rather left this as-is and refactor `BeatmapSetOverlay` in a separate pr, so we could leave the method type `OverlayHeader` and mark it as `[NotNull]` at the same time.
Because if we make this `Drawable.Empty()` we need to change the type to `Drawable` and if we want to call header - we will need to check if it's really the `OverlayHeader` and cast it to `T` every time

Reply:
i would have preferred to have this done in the backwards order (as in, deal with beatmap set overlay first, then unify) but i can barely bring myself to care at this point.

Reply:
sure!

Reply:
Fixed in https://github.com/ppy/osu/pull/11527/commits/65ece1aa722b74e6590b4b3d208a10141238164d commit